src/main.c

```diff
@@ -126,7 +126,7 @@ int main(int argc, char *argv[])
 	}
 
 	log_ell_enable(settings->log_level);
-
+	knot_cloud_set_log_priority("error");
```

Comment:
I think you should create a #define maybe to define those log priorities. What do you think?